packages/react-signature-frame/CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 All notable changes to this project will be documented in this file.
 
+## [@anvilco/react-signature-frame@1.7.0](https://github.com/anvilco/react-ui/compare/@anvilco/react-signature-frame@1.6.0...@anvilco/react-signature-frame@1.7.0) - 2022-03-31
```

Comment:
Does the changelog get updated when publishing? Like after this PR is merged?

Reply:
Changelog updates are part of `yarn build:changelog`